osgp/protocol-adapter-dlms/osgp-dlms/src/main/java/org/opensmartgridplatform/dlms/services/ObjectConfigService.java

```diff
@@ -27,16 +27,19 @@
 @Slf4j
 public class ObjectConfigService {
 
-  private List<DlmsProfile> dlmsProfiles;
+  private final List<DlmsProfile> dlmsProfiles = new ArrayList<>();
 
-  public ObjectConfigService() {}
+  ObjectConfigService() {}
 
+  /*
+   * Provide a list of DlmsProfile to the constructor or add a null value and profiles are loaded from the classpath resource '/dlmsprofiles'.
+   */
   public ObjectConfigService(final List<DlmsProfile> dlmsProfiles)
       throws ObjectConfigException, IOException {
     if (dlmsProfiles == null) {
-      this.dlmsProfiles = this.getDlmsProfileListFromResources();
+      this.dlmsProfiles.addAll(this.getDlmsProfileListFromResources());
     } else {
-      this.dlmsProfiles = dlmsProfiles;
+      this.dlmsProfiles.addAll(dlmsProfiles);
```

Comment:
Ik vind het zelf fraaier als in de constructor:
`this.dlmsProfiles = this.getDlmsProfileListFromResources();`

Reply:
Ja ga ik aanpassen.